http-server.ts

```diff
@@ -0,0 +1,11 @@
+import http = require("http");
+import {routeRequest} from './router'
```

Comment:
some of your quote marks are single, and some double. the standard is single,
add
```
"quotemark": [ true, "single", "avoid-escape", "avoid-template" ],
```
to tslint.json

Reply:
done